features/2025-07-radius-resource-types-contribution.md

```diff
@@ -0,0 +1,343 @@
+       - Integration Testing: Full integration with Radius CI/CD pipeline and release process
+       - Documentation: Complete user guides, troubleshooting, and best practices
+       - SLA Commitment: Defined support level and response time commitments
+       - Maintainer Review: Formal review and approval by Radius maintainers
+    
+    Contributor has to follow the contribution guidelines and the checklist for each stage to ensure that their contribution meets the requirements. Radius maintainers review the contribution meet the bar of quality for each of these stages before merging the contribution into the `resource-types-contrib` repository.
+
+- Should we ship all resource types in this repository as part of Radius ?
+
+  A Resource type cannot be useful without a Recipe. Every time a new resource type is added to the `resource-types-contrib` repository, it must be accompanied by at least one default Recipe that works with the resource type. Shipping just the resource type will impact the developer experience as it will cause deployment errors when there is no default Recipe available for the resource type. Based on the stages defined, we ship only resource types that have reached the `stable` stage as part of Radius. Community is encouraged to use the alpha and beta resource types and Recipes from the `resource-types-contrib` repository.
+
+- How can we ensure maintenance of community-contributed resource types?
+
+  - Radius maintainers triage bugs and issues reported in the `resource-types-contrib` repository. Based on the stages defined above, the Radius maintainers will prioritize bugs and issues reported for resource types and Recipes
+     - For bugs on existing resource types and Recipes, we follow the below process:
+        - If the resource type is in stable stage, the Radius maintainers will prioritize fixing these bugs and ensuring the resource types remain functional and up-to-date.
+        - If the resource type is in beta/ alpha stage, the contributor will be responsible for fixing the bug and ensuring the resource type remains functional and up-to-date. 
+    - For proposals on new resource types, we triage the proposal and assign it to the contributor who proposed the resource type if interested to contribute or leave it open for other contributors to pick up.
+
+- What level of technical support should be provided to contributors?
+   -  Detailed guidelines and templates for contributing resource types and Recipes
+   -  Clear documentation on testing and validation procedures
+   -  Radius Discord server contribution channel for community support and discussion
+
+- How can we incentivize contributions?
+  - Recognition through contributor showcases on the Radius website or documentation
+  - Swags for contributors who submit meaningful contributions
+  - Opportunities for contributors to become maintainers of the `resource-types-contrib` repository
```

Comment:
I know Radius is open-source but can we arrange Azure credits for contributions? Not for all of them but maybe first X of them...